README.md

```diff
@@ -139,7 +231,7 @@ pegging and using these $ATOMs to make voting decisions on Cosmos as a voting
 bloc. The following section describes the one and only way the AtomOne hub will
 use $ATOMs; by offering what is misleadingly referred to as "liquid staking".
 
-## Role as $ATOM "Liquid Staking" Provider
+### AtomOne's Strategy for $ATOM Bonding and Liquid Staking
```

Comment:
Was better before; not it’s inconsistent (mixing with and without AtomOne in the titles)